docs/platform/what-is-tizen/profiles/iot.md

```diff
@@ -0,0 +1,40 @@
+# Tizen IoT
+
+Tizen IoT profile allows developers to make various kinds of IoT devices and develop applications for the devices.
+Tizen IoT Headed and Headless binaries support any type of IoT devices based on the Linux kernel.
+
+## Development Environment for Tizen IoT
+Tizen IoT supports the following environment for development:
+- Hardware targets
+    - Raspberry Pi 3 and 4 as the official reference boards
+- Platform images
+    - Headless image for IoT devices without a display
+    - Headed image for IoT devices with a display
+
+## Developing Applications for Tizen IoT
+To develop applications for the IoT profile:
+- [Create Your First Tizen IoT .NET Application](../../../application/dotnet/get-started/iot/first-app.md)
```

Comment:
Just a suggestion:
Could we add this point as a single sentence and not as a single bullet:

To develop applications for the IoT profile, see [Create Your First Tizen IoT .NET Application](../../../application/dotnet/get-started/iot/first-app.md).

Please ignore if invalid.

Reply:
I think it would be better to have a consistent style with other pages (mobile and wearable) 
+ we could add more bullet points when more information gets available.